frontend/src/app/shared/slack-invite-box/slack-invite-box.widget.html

```diff
@@ -0,0 +1,35 @@
+<mat-card class="slack-dialog" #dialogContainer>
+  <mat-card-content class="content-container">
+    <h1>Join us on Slack!</h1>
+    <p>
+      Interested in becoming a part of the Undergraduate Computer Science
+      Experience Lab community?
+    </p>
+    <p>
+      Join the CSXL Slack to get updates about the XL, learn about new
+      opportunities for CS students, or meet your fellow CS majors!
+    </p>
+    <br />
+    <h3>
+      By accepting the below invite and joining the CSXL Slack, you are agreeing
+      to the following:
```

Comment:
Nit: Might be better if this header has a lower prominence! (like maybe `<h4>` or lower!)

Reply:
I'm going to take a stab at tweaking this language in a push.